conf/messages.en

```diff
@@ -736,15 +737,15 @@ amend.next.step.p2 = You can print a summary of your application on the next pag
 amend.site.continue = Accept and send
 amend-file-upload.help.linkText = Help with file types
 amend-file-upload.help.line1 = Each file must be 6MB or less.
-amend-file-upload.help.line2 = Accepted file formats
+amend-file-upload.help.line2 = Types of files you can upload
 
 supported-file-type.help.fileFormat1= Image (.jpeg, .jpg or .png)
 supported-file-type.help.fileFormat2= PDF (.pdf)
 supported-file-type.help.fileFormat3= MS Word document (.doc or .docx)
 supported-file-type.help.fileFormat4= MS Excel spreadsheet (.xls or .xlsx)
 supported-file-type.help.fileFormat5= MS PowerPoint (.ppt or .pptx)
 supported-file-type.help.fileFormat6= Open Document Format (.odt, .ods or .odp)
-supported-file-type.help.fileFormat7= Text files (.txt)
+supported-file-type.help.fileFormat7= Text files (.csv or .txt)
```

Comment:
this differs from the ticket
BUT
It looks right
Please clarify

Reply:
Yeah, it is different.  I thought the ticket was wrong so I queried it with Amrit.  I'll update the ticket.